Tests/BuildTests/IncrementalBuildTests.swift

```diff
@@ -145,4 +146,28 @@ final class IncrementalBuildTests: XCTestCase {
             XCTAssertNoMatch(log2, .contains("Planning build"))
         }
     }
+    // testing the fix for rdar://115777026
```

Comment:
I'd suggest writing out a comment with a high level description of the goal of the test (1 or 2 sentences), which will be more useful to readers instead of a rdar number.

Reply:
Done. Thank you @jakepetroules